homeassistant/components/buienradar/camera.py

```diff
@@ -95,7 +95,7 @@ def __init__(self, name: str, dimension: int, delta: float, country: str):
         # deadline for image refresh - self.delta after last successful load
         self._deadline: Optional[datetime] = None
 
-        self._unique_id = f"br_cam_{self._dimension}_{self._country}"
+        self._unique_id = f"{self._dimension}_{self._country}"
```

Comment:
This seem a bit iffy from an uniqueness perspective. There only ever exist one camera per country?

Reply:
It is not blocked to have multiple cameras per country with different dimensions. I'm have no idea how it is being used, so that's why at least I took it into the unique id, to be sure not to break anything.

Reply:
Joure right, I used to have 3 buienradar cameras for different regions in the country. Not sure if that is still possible (dont use them now) but would be great if it wasn't blocked/broken.

Reply:
So leave as is? My main problem with removing it is creating a breaking change, which I wouldn't want to do with such a small pr.

A breaking change is anyway coming with the config flow so it's better to address it there.